tests/cnf/core/network/metallb/tests/bgp-gracefulrestart.go

```diff
@@ -127,6 +144,157 @@ var _ = Describe("BGP Graceful Restart", Ordered, Label(tsparams.LabelGRTestCase
 		})
 
 	})
+
+	Context("with bfd", func() {
+		var (
+			bfdProfile   *metallb.BFDBuilder
+			bfdConfigMap *configmap.Builder
+		)
+
+		BeforeAll(func() {
+			By("Editing the MachineConfiguration CR named cluster to change node disruption policies")
+			updateMachineConfigurationNodeDisruptionPolicy()
+		})
+
+		BeforeEach(func() {
+			By("Creating BFD profile")
+			frrk8sPods, err := pod.List(APIClient, NetConfig.MlbOperatorNamespace, metav1.ListOptions{
+				LabelSelector: tsparams.FRRK8sDefaultLabel,
+			})
+			Expect(err).ToNot(HaveOccurred(), "Failed to list pods")
+			bfdProfile = createBFDProfileAndVerifyIfItsReady(frrk8sPods)
+
+			By("Creating MetalLb configMap")
```

Comment:
I suppose this is the frr.conf of the external BGP peer?

Reply:
changed